lesson_07/Robot/Run.java

```diff
@@ -0,0 +1,43 @@
+package lesson_07.Robot;
+
+import lesson_07.Robot.hands.SamsungHand;
+import lesson_07.Robot.hands.SonyHand;
+import lesson_07.Robot.hands.ToshibaHand;
+import lesson_07.Robot.heads.SamsungHead;
+import lesson_07.Robot.heads.SonyHead;
+import lesson_07.Robot.heads.ToshibaHead;
+import lesson_07.Robot.legs.SamsungLeg;
+import lesson_07.Robot.legs.SonyLeg;
+import lesson_07.Robot.legs.ToshibaLeg;
+
+public class Run{
+    public static void main(String[] args) {
+        /*
+        Создать по 3 реализации(Sony, Toshiba, Samsung) каждой запчасти(IHead, IHand, ILeg)
+        Класс SonyHead является примером реализацией головы от Sony.
+        Создайте 3 робота с разными комплектующими.
+        Например у робота голова и нога от Sony а, рука от Samsung.
+        У всех роботов вызовите метод action.
+        Среди 3-х роботов найдите самого дорогого.
+        */
+
+        Robot robot1 = new Robot(new SonyHead(1000), new ToshibaHand(550), new SamsungLeg(700));
```

Comment:
current implementations shows that price of part depends on constructor. It's better to have predefined parts prices and just collect robot from it